src/server/game/Movement/MotionMaster.cpp

```diff
@@ -479,11 +479,12 @@ void MotionMaster::MoveFall(uint32 id /*=0*/)
     if (std::fabs(_owner->GetPositionZ() - tz) < 0.1f)
         return;
 
+    _owner->AddUnitMovementFlag(MOVEMENTFLAG_FALLING);
```

Comment:
Just after my previous comment in the [Conversations] tab, I received an e-mail with a comment from Shauren, added to this point in the file, saying: "All changes made to this file are wrong. Creatures should never receive MOVEMENTFLAG_FALLING". Does that comment still hold true? (Maybe github somehow removed that comment and it should not have been removed? I don't know.)